MetadataInfo.json

```diff
@@ -0,0 +1,32 @@
+{
+    "Name": "Checker_Framework",
+    "DisplayName": "Checker Framework Demo",
+    "Version": "1.0",
+    "Email": "werner.dietl@uwaterloo.ca",
+    "SupportEmail": "z359chen@uwaterloo.ca",
+    "TermsOfUseUrl": "http://checkerframework.org/",
+    "PrivacyUrl": "http://checkerframework.org/",
+    "Institution": "University of Waterloo",
+    "InstitutionnstitutionUrl": "https://uwaterloo.ca/brand/sites/ca.brand/files/resize/uploads/images/universityofwaterloo_logo_vert_rev_rgb_1-300x195.png",
+    "InstitutionImageUrl": "http://openjml.cs.ucf.edu/images/jml-logo-small.png",
+    "MimeType": "text/plain",
+    "SupportsLanguageSyntax": null,
```

Comment:
why this value is `null`? According to the rise4fun description, it seems the value of `SupportsLanguageSyntax` should be `true` or `false`.